exoboots.py

```diff
@@ -0,0 +1,105 @@
+from opensourceleg.actuators.base import CONTROL_MODES
+from opensourceleg.actuators.dephy import DEFAULT_CURRENT_GAINS, DephyLegacyActuator
+from opensourceleg.robots.base import RobotBase
+from opensourceleg.sensors.base import SensorBase
+from opensourceleg.utilities import SoftRealtimeLoop
+
+from src.utils.actuator_utils import create_actuators
+from src.settings.constants import (
+    BAUD_RATE,
+    LOG_LEVEL,
+    FLEXSEA_FREQ,
+)
+
+
+class DephyExoboots(RobotBase[DephyLegacyActuator, SensorBase]):
```

Comment:
The structure looks good! One thing we can add to this class is the transmission ratio function (instead of the gear ration function mentioned in the OSL Actuator base class). When we call transmission_ratio() it will create a function for the transmission ratio based on the joint encoder. However, once we have completed the JIM testing we can directly do the torque to current conversion in the set_output_torque() function (which will also be a part of this class).